### DISCUSSION TITLE
Browserslist fails on build

### DISCUSSION THREAD

**Discussion by tmshv** [vercel/next.js#16685] [Help]
# Bug report

## Describe the bug

I am trying to use custom browserslist config with `supports es6-module` target.

```json
"browserslist": [
     "supports es6-module"
],
```

But running `next build` failes with error `BrowserslistError: Unknown browser query `ios_saf all`. Maybe you are using old Browserslist or made typo in query.`

```sh
> next build

info  - Creating an optimized production build
Failed to compile.

BrowserslistError: Unknown browser query `ios_saf all`. Maybe you are using old Browserslist or made typo in query.
```

## Expected behavior

```sh
$ npx browserslist "supports es6-module"
...
ios_saf 14.0
ios_saf 13.4-13.5
ios_saf 13.3
ios_saf 13.2
ios_saf 13.0-13.1
ios_saf 12.2-12.4
ios_saf 12.0-12.1
ios_saf 11.3-11.4
ios_saf 11.0-11.2
ios_saf 10.3
...
```

## System information

- OS: macOS
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 9.5.2]
- Version of Node.js: [e.g. 14.9.0]

**Comment by jamesmosier** [comment]
Hi @tmshv. Next supports a browerslist configuration [for autoprefixing CSS](https://nextjs.org/docs/advanced-features/customizing-postcss-config#customizing-target-browsers). Changing this will not effect the output of the JavaScript bundles. 

With that being said, what are you hoping to accomplish by having this custom browserlist config value?

**Comment by minusfive** [comment]
Can't speak for OP, but wanted to add to this as I'm hitting the same issue. I'm trying to cleanup my output as my app doesn't require support for older browsers. Using the new `supports` query (introduced in https://github.com/browserslist/browserslist/blob/master/CHANGELOG.md#413) seems like the cleaner option. I don't think Next.js consistently supports browserslist for all its configs, but it'd be great if it did, considering it's been babel's default for a while, and even webpack is moving in that direction (https://github.com/webpack/webpack/releases/tag/v5.0.0-rc.1).

Having a single `browserslist` setting in `package.json` to configure all plugins seems like a no-brainer, clean solution.
